src/__test__/utils/Cluster.ts

```diff
@@ -157,11 +157,11 @@ export class Cluster {
       if (this.retryCount > 0) {
         this.retryCount -= 1;
 
-        console.log(
-          `Failed to initialize cluster. Retry ${3 - this.retryCount}\n${
-            error.err
-          } `
-        );
+        // console.log(
+        //   `Failed to initialize cluster. Retry ${3 - this.retryCount}\n${
+        //     error.err
+        //   } `
+        // );
```

Comment:
There's various commented out debugging points in the file:
https://github.com/EventStore/EventStore-Client-NodeJS/blob/master/src/__test__/utils/Cluster.ts#L312
https://github.com/EventStore/EventStore-Client-NodeJS/blob/master/src/__test__/utils/Cluster.ts#L324
etc.

I should probably neaten it up by putting them behind a debug flag, rather than just uncommenting as needed.

Reply:
Made an issue for it:
https://github.com/EventStore/EventStore-Client-NodeJS/issues/228